config/eye.yml.erb.template

```diff
@@ -0,0 +1,69 @@
+<%% app_root = "/home/nucore/nucore.stage.tablexi.com/current" %>
+<%% rails_env = ENV["RAILS_ENV"] %>
+---
+name: <%= @app_name.underscore.camelize %> (<%%= rails_env %>)
```

Comment:
@jhanggi Is the quoting correct here? This might be pulled from the .tt file which converts these into actual values?